### PR TITLE
Upgrade Opensync version to 1.4.0

### PR DESCRIPTION
**Pull request by manigandanj**
Signed-off-by: manigandan.gopalakrishnan <manigandan.gopalakrishnan@ltts.com>

### CODE REVIEW

File: recipes-connectivity/opensync/opensync_1.4.0.bbappend [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VENDOR_URI = "git://git@github.com/rdkcentral/opensync-vendor-rdk-turris.git;protocol=ssh;branch=master;name=vendor;destsuffix=git/vendor/turris"
```
Comment (scthunderbolt) [inline recipes-connectivity/opensync/opensync_1.4.0.bbappend]: ssh protocol is not supported, please use https. ssh ports are frequently blocked by corporate firewalls.

Comment (manigandanj) [reply recipes-connectivity/opensync/opensync_1.4.0.bbappend]: thanks @scthunderbolt

File: recipes-connectivity/opensync/opensync_1.4.0.bbappend [hunk 2]
```diff
@@ -1,3 +1,3 @@
-VENDOR_URI = "git://git@github.com/rdkcentral/opensync-vendor-rdk-turris.git;protocol=ssh;branch=master;name=vendor;destsuffix=git/vendor/turris"
+VENDOR_URI = "git://git@github.com/rdkcentral/opensync-vendor-rdk-turris.git;protocol=https;branch=master;name=vendor;destsuffix=git/vendor/turris"
```
Comment (scthunderbolt) [inline recipes-connectivity/opensync/opensync_1.4.0.bbappend]: Please use the following variables in SRC_URI:
CMF_GITHUB_ROOT
CMF_GIT_PROTOCOL
CMF_GIT_MASTER_BRANCH

See https://github.com/rdkcentral/meta-turris/blob/master/recipes-ccsp/hal/hal-wifi-generic_git.bbappend for an example.

The variables are needed to make CMF branching and releases.

**Comment by scthunderbolt** [comment]
Build stalled when compiling open sync 1.4.0, this needs to be resolved before this can be merged.

**Comment by scthunderbolt** [comment]
Compilation is stuck at:

> 14902 jenkins   20   0      12     12      0 R  99.0  0.0  18:47.67 genconfig

taking 99% of CPU for 30+ mins and still going:

> Currently  1 running tasks (4029 of 4046)  99% 
> 0: opensync-1.4.0-r1 do_compile - 1813s (pid 11171)

**Comment by manigandanj** [comment]
Closing this PR since i created fresh Pull Request https://github.com/rdkcentral/meta-turris/pull/97 with changes requested in this PR.

**Pull request closed without merging**